core/src/extcmd/command_request.cc

```diff
@@ -36,8 +36,10 @@ using namespace com::centreon::broker::extcmd;
  *  Default constructor.
  */
 command_request::command_request()
-  : uuid(QUuid::createUuid().toString().toStdString()),
-    with_partial_result(false) {}
+  : with_partial_result(false) {
+  misc::Uuid tmp_uuid;
+  uuid = tmp_uuid.to_string();
+}
```

Comment:
Itn't it possible to call directly something like uuid{misc::Uuid().to_string()} before the {} block?

Reply:
done